Flow.Launcher/MainWindow.xaml

```diff
@@ -81,7 +82,7 @@
                      AllowDrop="True"
                      Visibility="Visible"
                     Background="Transparent"
-                    Margin="18,0,56,0">
+                    Margin="18,1,0,1" HorizontalAlignment="Left" Width="660">
```

Comment:
what's this change for?

Reply:
hmmm that's not intended, I will remove it.

Reply:
could we remove this one please

Reply:
sure